packages/toolpad-utils/src/collections.ts

```diff
@@ -93,3 +95,66 @@ export function filterKeys<U>(
 ): Record<string, U> {
   return mapProperties(obj, ([key, value]) => (filter(key) ? [key, value] : null));
 }
+
+/**
+ * Check if two values are equal - that is,
```

Comment:
The description seems cut off after "that is,"?
If you copied this function or any large parts of it from somewhere, we should reference the original source.